### PR TITLE
Edit the What is Istio file for clarity.

### PR DESCRIPTION
**Pull request by rcaballeromx**
This edit includes:

* The rewrite of all instances of passive voice
* The simplification of complex sentence structures
* The splitting of run-on-sentences
* The disambiguation of relative pronouns
* The removal of conjunctives such as should, could, and would
* The replacement of long series with lists
* The improvement of punctuation
* The addition of emphatic markup including links

The purpose behind the edits is to improve content flow and allow easier
consumption for international audiences. The simpler structures will also have a
positive impact in the translation.

Signed-off-by: rcaballeromx <grca@google.com>

### CODE REVIEW

File: content/docs/concepts/what-is-istio/index.md [hunk 1]
```diff
@@ -7,54 +7,91 @@ aliases:
...
+achieves it without requiring any changes in service code. To add Istio
+support to services, you deploy a special sidecar proxy throughout your
+environment. The proxy intercepts all network communication between
+microservices. You can configure and manage the proxy using Istio's control
+plane functionality.
+
+Istio currently supports:
+
+* Service deployment on Kubernetes
+* Services registered with Consul or Eureka
```
Comment (geeknoid) [inline content/docs/concepts/what-is-istio/index.md]: Remove Eureka, we're deprecating this.

File: content/docs/concepts/what-is-istio/index.md [hunk 2]
```diff
@@ -7,54 +7,91 @@ aliases:
...
 ## Why use Istio?
 
-Istio addresses many of the challenges faced by developers and operators as monolithic applications transition towards a distributed microservice architecture. The term **service mesh** is often used to describe the network of
-microservices that make up such applications and the interactions between them. As a service mesh grows in size and complexity, it can become harder to understand
-and manage. Its requirements can include discovery, load balancing, failure recovery, metrics, and monitoring, and often more complex operational requirements
-such as A/B testing, canary releases, rate limiting, access control, and end-to-end authentication.
+Istio addresses the challenges developers and operators face as monolithic
+applications transition towards a distributed microservice architecture.
+
+The term **service mesh** is used to describe the network of microservices that
```
Comment (geeknoid) [inline content/docs/concepts/what-is-istio/index.md]: Given that "service mesh" is two words, is it correct to call that a "term"?

**Comment by geeknoid** [comment]
Note the circleci warnings:

content/docs/concepts/what-is-istio/index.md:50: MD004 Unordered list style
content/docs/concepts/what-is-istio/index.md:54: MD004 Unordered list style
content/docs/concepts/what-is-istio/index.md:59: MD004 Unordered list style
content/docs/concepts/what-is-istio/index.md:66: MD004 Unordered list style
content/docs/concepts/what-is-istio/index.md:71: MD004 Unordered list style
content/docs/concepts/what-is-istio/index.md:86: MD004 Unordered list style
content/docs/concepts/what-is-istio/index.md:92: MD004 Unordered list style
content/docs/concepts/what-is-istio/index.md:182: MD004 Unordered list style
content/docs/concepts/what-is-istio/index.md:196: MD004 Unordered list style
content/docs/concepts/what-is-istio/index.md:206: MD004 Unordered list style
content/docs/concepts/what-is-istio/index.md:213: MD004 Unordered list style

This is telling that the markdown file should consistently use either - or * to create lists, pick one uniformly for the whole doc.

**Comment by rcaballeromx** [comment]
@geeknoid  noted and fixed.

File: content/docs/concepts/what-is-istio/index.md (hunk 2 above)
Comment (rcaballeromx) [reply content/docs/concepts/what-is-istio/index.md]: Yes, terms can be compound terms which are composed of more than one word.

File: content/docs/concepts/what-is-istio/index.md (hunk 1 above)
Comment (rcaballeromx) [reply content/docs/concepts/what-is-istio/index.md]: Understood.

**Comment by geeknoid** [comment]
I just merged a change to this same file. I realized after clicking Submit that this would cause a conflict with your change.

I basically took a bit of content from the about/intro/index.md file and pasted it at the top of the what-is-istio file, then deleted about/intro/index.md

You'll need to do "git pull upstream" followed by "git rebase upstream/master". This will result in merge conflicts. You then need to load the index.md file and look for >>> markers. Fix things up the way you'd like, and then go back to the command-line. Then do "git add ." followed by "git rebase --continue". Finally, push your PR up to GitHub again.

Sorry about that.

**Comment by rcaballeromx** [comment]
@geeknoid No problem, I'll update my fork and solve the conflict locally before rebasing and pushing again. Thanks for the heads up.

**Comment by rcaballeromx** [comment]
@geeknoid All fixed! It took some Git magic but the conflicts are resolved with the best of both worlds.

**Pull request merged**